### ISSUE TITLE
When clicking a dot it is kept selected until i click somewhere else

### ISSUE THREAD

**Issue by Kerberos-666** [akiran/react-slick#1745]
### Guidelines for posting a new issue

* Please replicate your issue with this [CodeSandBox](https://codesandbox.io/s/ppwkk5l6xx) and provide a link to it along with the issue description

**Comment by the-J** [comment]
True, have the same issue. Did you find any solution? @Kerberos-666

**Comment by Kerberos-666** [comment]
Nope, couldn't find any @the-J

**Comment by gautamz07** [comment]
@Kerberos-666 and @the-J 

The happens because, the styles for `.active` and `:focus` , ':hover' are the same , so just change the style for :focus and :active in you're style like so:

```
.slick-dots li button:hover:before, .slick-dots li button:focus:before {
    opacity: 0.25;
}
```

**Comment by wangjiangs** [comment]
I have the same issue, but I resolved by this code, it worked for me :
`dom.addEventLisener('mousedown', () => window.getSelection().empty())`
